src/reference/asciidoc/changes-4.0-4.1.adoc

```diff
@@ -12,50 +12,50 @@ Version 4.1 added a number of new components.
 ===== Promise<?> Gateway
 
 The messaging gateway methods now support a Reactor `Promise` return type.
-See "`<<async-gateway>>`".
+See <<async-gateway>>.
 
 [[x4.1-web-socket-adapters]]
 ===== WebSocket support
 
 The `WebSocket` module is now available.
 It is fully based on the Spring WebSocket and Spring Messaging modules and provides an `<inbound-channel-adapter>` and an `<outbound-channel-adapter>`.
-See "`<<web-sockets>>`" for more information.
+See <<web-sockets>> for more information.
 
 [[x4.1-scatter-gather]]
 ===== Scatter-Gather Enterprise Integration Pattern
 
 We implemented the scatter-gather enterprise integration pattern.
-See "`<<scatter-gather>>`" for more information.
+See <<scatter-gather>> for more information.
 
 [[x4.1-Routing-Slip]]
 ===== Routing Slip Pattern
 
 We added the routing slip EIP pattern implementation.
-See "`<<routing-slip>>`" for more information.
+See <<routing-slip>> for more information.
 
 [[x4.1-idempotent-receiver]]
 ===== Idempotent Receiver Pattern
 
 We added the idempotent receiver enterprise integration pattern implementation by adding the `<idempotent-receiver>` component in XML or the `IdempotentReceiverInterceptor` and `IdempotentReceiver` annotations for Java configuration.
-See "`<<idempotent-receiver>>`" and the https://docs.spring.io/spring-integration/api/index.html[Javadoc] for more information.
+See <<idempotent-receiver>> and the https://docs.spring.io/spring-integration/api/index.html[Javadoc] for more information.
 
 [[x4.1-BoonJsonObjectMapper]]
 ===== Boon `JsonObjectMapper`
 
 We added T\the Boon `JsonObjectMapper` for the JSON transformers.
```

Comment:
Here is some typo. Probably not yours, but I'd appreciate if you fix it in this PR. 😄 
Thanks